recipe/meta.yaml

```diff
@@ -31,15 +31,13 @@ source:
 
 build:
   number: 4
-  skip: true  # [win or osx]
   features:
     - vc9  # [win and py27]
     - vc10  # [win and py34]
     - vc14  # [win and py35]
 
 requirements:
   build:
-    # HACK: this is here to get Windows features (vc??) to work.
     - python  # [win]
```

Comment:
`conda-smithy` needs that to get the matrix correctly. But we don't need that for recipes that will not use conda-forge machinery. (And we are looking into updating `conda-smithy` so that is no longer needed.)